request_processor.py

```diff
@@ -40,3 +44,15 @@ async def request_handling(
             ) from e
     res_body = output_type(**output) if output_type else output
```

Comment:
let's try and keep it simple
add wrap_output as optional parameter in the function and make it default to None
remove the decorator
remove the new functions


```suggestion
    res_body = output_type(**output) if output_type else output
    if wrap_output:
        res_body = {
            'data': res_body,
            'message': "Request received.",
            'request_id': "req-" + str(uuid.uuid4())
        }
```